pallets/fruniques/src/lib.rs

```diff
@@ -280,11 +282,19 @@ pub mod pallet {
 			attributes: Option<Attributes<T>>,
 			parent_info_call: Option<ParentInfoCall<T>>,
 		) -> DispatchResult {
+			//Ensure the collection exists
 			ensure!(Self::collection_exists(&class_id), Error::<T>::CollectionNotFound);
+			// Ensure the user is in the collection
 			let user: T::AccountId = ensure_signed(origin.clone())?;
-			Self::is_authorized(user.clone(), class_id, Permission::Mint)?;
```

Comment:
The initially requested change was because the rbac pallet returns the `NoAuthorized` error,  which makes sense because if the user has not been invited to the collection, then they are not assigned to a role within that scope (collectionID).
The other possible case is when the Frunques pallet isn't initialized. Since the pallet is only initialized once in a lifetime, it is easy to rule out that the error is due to the fact that the pallet was not initialized.
 You could create a new function that checks if the user is part of a collection using the `has_role` function from the rbac. Depending on which order you put all of them together, it will be the error shown in the UI. My final thought is: Create a helper function to check if the user is in the collection.